app/src/main/java/com/koalatea/sedaily/feature/commentList/epoxy/CommentEpoxyModelWithHolder.kt

```diff
@@ -9,15 +11,21 @@ import com.koalatea.sedaily.R
 abstract class CommentEpoxyModelWithHolder : BaseCommentEpoxyModelWithHolder<CommentHolder>() {
 
     @EpoxyAttribute lateinit var replyClickListener: () -> Unit
+    @EpoxyAttribute lateinit var upVoteClickListener: () -> Unit
+
 
     override fun bind(holder: CommentHolder) {
         super.bind(holder)
 
         holder.replayButton.setOnClickListener { replyClickListener() }
+        holder.upVoteButton.setOnClickListener { upVoteClickListener() }
+        holder.upVoteButton.isSelected = upvoted
+        holder.upVoteButton.text = score.toString()
```

Comment:
That is unsafe, please double check that score is > 0 and display nothing otherwise. Displaying 0 will not really add much value

Reply:
I've fixed this for comments and replies.